gclient/src/api/mod.rs

```diff
@@ -30,12 +30,35 @@ pub struct GearApi(Signer);
 
 impl GearApi {
     pub async fn init(address: WSAddress) -> Result<Self> {
+        Self::init_with(address, "//Alice").await
+    }
+
+    // Suri is Substrate URI which identifies user with mnemonic
+    // or provides defaults from keyring: e.g. "//Alice".
+    //
+    // Password for URI should be specified in the same str, separated with ':'.
+    pub async fn init_with(address: WSAddress, suri: impl AsRef<str>) -> Result<Self> {
+        let mut suri = suri.as_ref().split(':');
```

Comment:
I don't know whether `:` is acceptable char in the password, but `splitn` can avoid this problem.
```suggestion
        let mut suri = suri.as_ref().splitn(2, ':');
```